index.js

```diff
@@ -511,6 +522,17 @@ module.exports = {
     return true;
   },
 
+  _emberDataVersionRequiresPackagesPolyfill() {
+    let dataAddon = this.project.addonPackages['ember-data'];
```

Comment:
What is `this.project.addonPackages`? Do you mean `this.project.findAddonByName`?

Reply:
I think you may also need to check for this transitively though, e.g. if the project itself does not depend on ember-data but some other shared addon does...

Reply:
@rwjblue recommended pattern for that detection? similar to what we do in ember-m3?

Reply:
or rather what we do in ember-data: https://github.com/emberjs/data/blob/master/packages/private-build-infra/src/packages.js#L5